docs/manual/docs/cmd/user/user-get.md

```diff
@@ -0,0 +1,49 @@
+# spo user get
+
+Gets a site user within specific web
+
+## Usage
+
+```sh
+spo user get [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the web to get the user within 
+`--email [email]`|Email of the user to retrieve information for. Use either "email", "id" or "loginName", but not all. e.g 'john.doe@mytenant.onmicrosoft.com'
```

Comment:
Doesn't match the order in which the options are listed in the command's help. ID is listed first